### ISSUE TITLE
getting segmentation fault after starting/stopping processes, possibly after hibernation

### ISSUE THREAD

**Issue by mateodelnorte** [electronifie/node-quickfix#3]
From time to time I get a segfault either after starting and stopping processes multiple times, or after reawaking form hibernate/sleep. 
```
make[1]: *** [run-debug] Segmentation fault: 11
make: *** [run] Error 2
```
Deleting all files within the /data directory (`rm -rf data/`) for both processes involved fixes the issue. The files may be getting corrupted in some way.

**Comment by johannbotha** [comment]
I've seen this problem as well, but I have found different solutions. Try doing a `make reset-queues` or look for duplicate fix-routers .

**Comment by stefankutko** [comment]
We believe we have fixed this with version 0.2.2.  Will test out shortly and confirm.
